lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -3243,8 +3243,16 @@ namespace Js
         }
 #endif
         Assert(((LoopEntryPointInfo*) entryPointInfo)->loopHeader == loopHeader);
-        Assert(reinterpret_cast<void*>(entryPointInfo->jsMethod) == nullptr);
-        entryPointInfo->jsMethod = entryPoint;
+        Assert(entryPointInfo->jsMethod == nullptr);
+        entryPointInfo->jsMethod = (void*)entryPoint;
```

Comment:
Ah.. conflict resolve issue. Don't know why it didn't show up in my diff. Thanks for catching.

---
In reply to: [65243518](https://github.com/Microsoft/ChakraCore/pull/1029#discussion_r65243518) [](ancestors = 65243518)